spec/models/task_spec.rb

```diff
@@ -1,5 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  let(:task) { create(:task) }
+
+  # titleが存在しなければ無効であること
+  it "is invalid without a title" do
+    task = Task.new(title: nil, status: 'doing')
+    expect(task).to be_invalid
+  end
+
+  # titleの値が重複していれば無効であること
+  it "is invalid with a duplicate task's title" do
+    task
+    duplicated_task = Task.create(title: 'テストタイトル', status: 'todo')
+    duplicated_task.valid?
+    expect(duplicated_task.errors[:title]).to include("has already been taken")
+  end
+
+  # statusが存在しなければ無効であること
+  it "is invalid without a status" do
+    task = Task.new(title: 'テストタイトル', status: nil)
+    task.invalid?
+    expect(task.errors[:status]).to include("can't be blank")
+  end
+
+  # 項目に過不足がないデータはバリデーションエラーとならずに正常に作成できること
+  it "is valid when it has a sufficient value" do
+    task
+    task.invalid?
```

Comment:
この`invalid?`の結果が`false`であることは`expect`として確認しなくても大丈夫ですか？
```suggestion
    expect(task).to be_valid
```

Reply:
5b028c7 にて修正しました。